src/Symfony/Bundle/FrameworkBundle/Tests/Controller/TemplateControllerTest.php

```diff
@@ -39,4 +43,22 @@ public function testNoTwig()
         $controller->templateAction('mytemplate')->getContent();
         $controller('mytemplate')->getContent();
     }
+
+    public function testContext()
+    {
+        $templateName = 'template_controller.html.twig';
+        $context = [
+            'param' => 'hello world',
+        ];
+        $expected = '<h1>'.$context['param'].'</h1>';
+
+        $loader = new ArrayLoader();
+        $loader->setTemplate($templateName, '<h1>{% block title %}{{param}}{% endblock %}</h1>');
```

Comment:
for short `'<h1>{{ param }}</h1>'` ? the block tag is not really needed in this test case.